Makefile

```diff
@@ -124,6 +124,22 @@ pjsua-test: cmp_wav
 cmp_wav:
 	cd tests/pjsua/tools && make
 
+fuzz:
```

Comment:
You're not supposed to modify build.mak.in for non-global settings. Instead, you can put it in `tests/fuzz/Makefile`. Then, here,  you can do similar like above, i.e. call `cd tests/fuzz && make`.